components/RadioButtons/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableHighlight, View } from 'react-native';
+import { StyleSheet } from 'react-native';
+
+export default class RadioButtons extends React.Component {
+  static propTypes = {
+    buttonNames: PropTypes.arrayOf(PropTypes.string).isRequired,
+    input: PropTypes.shape({
+      value: PropTypes.string.isRequired,
+      onChange: PropTypes.func.isRequired,
+    }),
+  }
+
+  render() {
+    const { input: { value, onChange } } = this.props;
```

Comment:
might as well put `buttonNames` in here so later you can just call it as `buttonNames` instead of `this.props.buttonNames`